db/flush_job.cc

```diff
@@ -54,6 +54,17 @@
 
 namespace rocksdb {
 
+const char* flush_reason_map[] = {"Unknown",
```

Comment:
maybe make a `GetFlushReasonString()` function with a switch statement on the enum value? That way the compiler can force us to keep the strings and enum values in-sync. And we wouldn't have to worry about keeping their entries in the same order.